bin/elixir.ps1

```diff
@@ -0,0 +1,379 @@
+      break
+    }
+
+    "--logger-otp-reports" {
+      $private:TempVal = $Args[$i + 1]
+      if ($TempVal -in @("true", "false")) {
+        $ErlangParams.AddRange([string[]]@("-logger", "handle_otp_reports", $Args[++$i]))
+      }
+      break
+    }
+
+    "--logger-sasl-reports" {
+      $private:TempVal = $Args[$i + 1]
+      if ($TempVal -in @("true", "false")) {
+        $ErlangParams.AddRange([string[]]@("-logger", "handle_sasl", $Args[++$i]))
+      }
+      break
+    }
+
+    "--erl" {
+      $BeforeExtras.Add($Args[++$i])
+      break
+    }
+
+    "--werl" {
+      if ($IsWindows) {
+        $ErlExec = "werl"
+      }
+      break
+    }
+
+    "+iex" {
+      $ElixirParams.Add("+iex")
+      $UseIex = $true
+
+      break
+    }
+
+    "+elixirc" {
+      $ElixirParams.Add("+elixirc")
+      break
+    }
+
+    "--rpc-eval" {
+      $private:Key = $Args[++$i]
+      $private:Value = $Args[++$i]
+
+      if ($null -eq $Key) {
+        Write-Error "--rpc-eval: NODE must be present"
+        exit 1
+      }
+
+      if ($null -eq $Value) {
+        Write-Error "--rpc-eval: COMMAND for the '$Key' node must be present"
+        exit 1
+      }
+
+      $ElixirParams.Add("--rpc-eval")
+      $ElixirParams.Add($Key)
+      $ElixirParams.Add("$(QuotedString $Value)")
+      break
+    }
+
+    "--boot-var" {
+      $private:Key = $Args[++$i]
+      $private:Value = $Args[++$i]
+
+      if ($null -eq $Key) {
+        Write-Error "--boot-var: VAR must be present"
+        exit 1
+      }
+
+      if ($null -eq $Value) {
+        Write-Error "--boot-var: Value for the '$Key' var must be present"
+        exit 1
+      }
+
+      $ElixirParams.Add("-boot_var")
+      $ElixirParams.Add($Key)
+      $ElixirParams.Add("$(QuotedString $Value)")
+      break
+    }
+
+    "--pipe-to" {
+      $RunErlPipe = $Args[++$i]
+      $RunErlLog = $Args[++$i]
+
+      if ($null -eq $RunErlPipe) {
+        Write-Error "--pipe-to: PIPEDIR must be present"
+        exit 1
+      }
+
+      if ($null -eq $RunErlLog) {
+        Write-Error "--pipe-to: PIPELOG must be present"
+        exit 1
+      }
+
+      if ($RunErlPipe.EndsWith("/") -or $RunErlLog.EndsWith("/")) {
+        Write-Error "--pipe-to: PIPEDIR and PIPELOG must not end with a slash"
+        exit 1
+      }
+
+      break
+    }
+
+    Default {
+      $private:Normalized = NormalizeArg $Arg
+      if ($Normalized.StartsWith("-")) {
+        $AllOtherParams.Add($Normalized)
+      }
+      else {
+        $AllOtherParams.Add("$(QuotedString $Normalized)")
+      }
+      break
+    }
+  }
+}
+
+# Support for ANSI is only disable if TERM or NO_COLOR env vars are set.
+# This will change the $PSStyle.OutputRendering property.
+if ($PSStyle.OutputRendering -ne "PlainText") {
+  # TODO: find a way to detect if the term is interactive on Windows.
+  if ($IsWindows -or (test -t 1 -a -t 2)) {
+    $BeforeExtras.InsertRange(0, [string[]]@("-elixir", "ansi_enabled", "true"))
+  }
+}
```

Comment:
We can remove this. From Erlang/OTP 26, the VM detects this. And we can say initially PS1 support is for OTP 26+.

```suggestion
```